sourcegraph/tests/localDevMode_test.yaml

```diff
@@ -0,0 +1,17 @@
+suite: localDevMode
+templates:
+  - frontend/sourcegraph-frontend.Deployment.yaml
+tests:
+  - it: should not have a resource block when localDevMode=true
```

Comment:
This could get quite verbose for extended tests, but it's readable and easy to expand. It would be ideal if we could pass in an entire values.yaml instead of specifying keys directly here, but https://github.com/quintush/helm-unittest/issues/132 is unanswered so maybe that means we can't.

The [snapshot testing feature](https://github.com/quintush/helm-unittest#snapshot-testing) is interesting - it might be a way we can get a more comprehensive test for "free" (the testing would be more fragile against intentional changes).

Reply:
I think this is what we're needing for more complex tests?

``` 
 -v, --values stringArray     absolute or glob paths of values files location, default no values files
```